### PR TITLE
Spread working directory

### PR DESCRIPTION
**Pull request by PedroHLC**

### CODE REVIEW

**Comment by PedroHLC** [comment]
Not sure if this will work, the idea is to:
```
     - uses: PaackEng/elm-lint-action@main
       with:
         compile-working-directory: 'showcase'
         extra-path: '../src'
```

**Comment by PedroHLC** [comment]
Just tested it here, and it seems to have worked:
https://github.com/PaackEng/paack-ui/runs/1832239723

File: action.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name: 'Elm Validate'
 description: 'Runs Elm through elm-format and elm-review then compiles it'
 compile-working-directory:
-    description: 'Working directory of the compile step'
+    description: 'Working directory of linter/review/compile step'
```
Comment (renanpvaz) [inline action.yml]: why not just `working-directory` then? Also, small but "lint"/review/compile makes more sense

**Review by renanpvaz — APPROVED** [review]

File: action.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
     required: false
-    default: 'src'
+    default: '.'
```
Comment (PedroHLC) [inline action.yml:6]: Just to be registered in the PR, looks like this default does not makes difference at all. Changing it did not change any behavior, as seen in https://github.com/PaackEng/wms-web/actions/runs/537721028

File: action.yml (hunk 1 above)
Comment (PedroHLC) [reply action.yml]: > why not just working-directory then?

As it's a reserved word, most probably, that will affect the entire action and not just those steps. And we want `yarn install` to run from the project's root.

**Pull request merged**